config/config.go

```diff
@@ -502,13 +501,6 @@ type PlanCache struct {
 	Shards   uint `toml:"shards" json:"shards"`
 }
 
-// PreparedPlanCache is the PreparedPlanCache section of the config.
-type PreparedPlanCache struct {
-	Enabled          bool    `toml:"enabled" json:"enabled"`
-	Capacity         uint    `toml:"capacity" json:"capacity"`
-	MemoryGuardRatio float64 `toml:"memory-guard-ratio" json:"memory-guard-ratio"`
-}
```

Comment:
If we remove it from the config file, how to keep compatible when upgrading?

Reply:
Because it is in the deprecated list, it will print a warning saying that the option specified in the config file is no longer supported. We will have to add in the release notes that the previously supported config option is now available as a system variable.